sound/soc/sof/intel/hda-ipc.c

```diff
@@ -234,19 +230,20 @@ irqreturn_t hda_dsp_ipc_irq_handler(int irq, void *context)
 {
 	struct snd_sof_dev *sdev = context;
 	int ret = IRQ_NONE;
+	u32 irq_status = 0;
```

Comment:
@keyonjie do we need to intialize this to 0? we had to remove a lot of these lately. so we should be critical of adding these in the future too.

Reply:
Good catch, this init is not needed?

Reply:
not needed, let me remove it.